### PR TITLE
Updates for RMW version 0.7.1

### PR DESCRIPTION
**Pull request by alsora**
Work in progress PR to address https://github.com/ros2/rmw_dps/issues/6

### CODE REVIEW

File: rmw_dps_cpp/src/type_support_common.hpp [hunk 1]
```diff
@@ -67,8 +68,15 @@ _create_type_name(
...
-  return
-    std::string(members->package_name_) + "::" + sep + "::dps_::" + members->message_name_ + "_";
+
+  std::ostringstream ss;
+  std::string message_namespace(members->message_namespace_);
+  std::string message_name(members->message_name_);
+  if (!message_namespace.empty()) {
+    ss << message_namespace << "::";
+  }
+  ss << "dds_::" << message_name << "_";
```
Comment (malsbat) [inline rmw_dps_cpp/src/type_support_common.hpp]: Can you change this to sep << "::dps_" ...?

**Comment by malsbat** [comment]
Looks good, thanks, just one small fixup.

File: rmw_dps_cpp/src/type_support_common.hpp (hunk 1 above)
Comment (alsora) [reply rmw_dps_cpp/src/type_support_common.hpp]: I changed `dds_` into `dps_`.
However, the separator is not necessary anymore (in the last commit I also removed it from the arguments).

See here for the field `message_namespace`
https://github.com/ros2/rosidl/commit/176bbe2640983e74fd1104017a9e153f089eef21

Comment (malsbat) [reply rmw_dps_cpp/src/type_support_common.hpp]: Great, thanks for the clarification

**Pull request merged**